scripts/export.py

```diff
@@ -119,6 +122,26 @@ def load_controls(loaded_rules: dict):
             logging.error(f"failed to open control: '{path_in_str}'")
             raise TypeError(e)
         
+        categoryFieldName = "category"
+        subCategoryFieldName = "subCategory"
```

Comment:
Add as consts to top of file

Reply:
why do you think it should be on the top of the file?